### PR TITLE
etcd-on-ramfs: add openstack reviewers

### PR DESCRIPTION
**Pull request by EmilienM**
Both of our teams contributed to it, let's maintain it together.

### CODE REVIEW

**Comment by mdbooth** [comment]
@EmilienM I generated the requested metadata to fix the test failure and pushed.

**Review by sandrobonazzola — APPROVED** [review]
/lgtm

File: ci-operator/step-registry/ipi/conf/etcd/on-ramfs/OWNERS [hunk 1]
```diff
@@ -1,4 +1,6 @@
 approvers:
+- openstack-approvers
 - ovirt-approvers
 reviewers:
+- openstack-approvers
```
Comment (mandre) [inline ci-operator/step-registry/ipi/conf/etcd/on-ramfs/OWNERS:5]: nit: we have an `openstack-reviewers` group, we should use it.

**Review by mandre — COMMENTED** [review]
/lgtm

**Pull request merged**